app/components/views/GovernancePage/Proposals/ProposalList.js

```diff
@@ -3,27 +3,46 @@ import { activeVoteProposals, preVoteProposals, votedProposals, proposals } from
 import { VotingProgress } from "indicators";
 import { PoliteiaLoading, NoProposals } from "indicators";
 
-const ProposalListItem = ({ name, timestamp, token, voting, voteCounts, tsDate, onClick }) => (
-  <div className="proposal-list-item" onClick={() => onClick(token)}>
+const VoteChoice = ({ currentVoteChoice }) =>
+  <div className={"proposal-vote-choice " + currentVoteChoice}/>;
+const VoteResults = ({ currentVoteChoice, quorumPass, voteResult }) => (
+  <div className="proposal-vote-result">
+    <div className="proposal-vote-choice-area">
+      <div className={"proposal-vote-choice " + currentVoteChoice}/>
+      <div className="proposal-vote-choice-text">{currentVoteChoice !== "abstain" ? <Aux><T id="proposal.voted" m="Voted"/> {currentVoteChoice}</Aux> : <T id="proposal.noVote" m="No vote cast"/>}</div>
+    </div>
+    <div className="proposal-vote-passage">{quorumPass ? voteResult : <T id="proposals.quorumNotMet" m="Quorum not met"/>}</div>
+  </div>
+);
+
+const ProposalListItem = ({ name, timestamp, token, voteCounts, tsDate, onClick, voteStatus, currentVoteChoice, quorumPass, voteResult }) => (
+  <div className={"proposal-list-item " + voteResult} onClick={() => onClick(token)}>
     <div className="info">
       <div className="proposal-name">{ name }</div>
       <div className="proposal-token">{ token }</div>
+      {voteStatus !== 4 &&
```

Comment:
Don't use hard coded numbers. Import the `VOTESTATUS_` constants from `GovernanceActions.js` and use those instead.